src/pages/dashboard.tsx

```diff
@@ -281,14 +283,18 @@ export default function DashboardPage(props: PageProps) {
         </main>
       </div>
 
-      {/*{parsedAnnouncements[0].id !== lastReadAnnouncement && (*/}
-      {/*  <div className="h-12">*/}
-      {/*    <AnnouncementBanner*/}
-      {/*      announcement={parsedAnnouncements[0]}*/}
-      {/*      onDismiss={() => setLastReadAnnouncement(parsedAnnouncements[0].id)}*/}
-      {/*    />*/}
-      {/*  </div>*/}
-      {/*)}*/}
+      {bannerIsOpen && userSettings.totalNoVisits > 0 && (
+        <div className="h-12">
+          <AnnouncementBanner
+            announcement={{
+              date: '2020',
+              id: 'total-number-of-visits',
+              title: 'Total number of Visits: ' + userSettings.totalNoVisits,
+            }}
+            onDismiss={() => setIsBannerOpen(false)}
+          />
+        </div>
+      )}
```

Comment:
what it was like before is fine... just add `userSettings.totalNoVisits > 10` or something in addition to the other condition